src/lib.rs

```diff
@@ -22,26 +24,24 @@
 //!     0., 0., 0., 1., 1., 1., 0., 0., 0., 0.,
 //!     0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,
 //!     0., 0., 0., 0., 0., 0., 0., 0., 0., 0.
-//! ], &[0.5]); // values, thresholds
-//! ```
-//! __*Output:*__
-//! ```text
-//! [Feature {
-//!   bbox: None,
-//!   geometry: Some(Geometry {
-//!     bbox: None,
-//!     value: MultiPolygon([[[
+//! ], &[0.5]).unwrap(); // values, thresholds
+//!
+//! let output = serde_json::json!({
```

Comment:
I made this example executable, and changed the output to use the `serde_json::json!` macro which is a little more intuitive to read IMO because it reads like "normal" json rather than the somewhat bespoke structs from the geojson crate.

Reply:
:+1: